agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -243,13 +243,23 @@ private static void logWarningIfUsingInternalAttributes(String attributeKey) {
   }
 
   private static void overlayProfilerEnvVars(Configuration config) {
+    if (isOpenJ9Jvm()) {
+      configurationLogger.warn(
+          "Profiler is not supported for an OpenJ9 JVM. Instead, please use an OpenJDK JVM.");
+      return;
+    }
```

Comment:
this won't update `profiler.enabled` if it was set via boolean, how about moving this check (and updating the boolean) near the end of `create` method?

Reply:
The `profiler.enabled` will keep its default value, that is to say false. I would prefer to have all the code related to  `profiler.enabled`  intialization in only one place.